### PR TITLE
update install and troubleshoot powerflex to do helm only install

### PR DESCRIPTION
**Pull request by nb950**
# Description
update install and troubleshoot powerflex to do helm only install

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| [dell/csm#85](https://github.com/dell/csm/issues/85)|

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [x] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/installation/helm/powerflex.md [hunk 1]
```diff
@@ -223,7 +223,7 @@ kubectl create -f deploy/kubernetes/snapshot-controller
 | image | image for podmon. | No | " " |
 
 
-10. Install the driver using `csi-install.sh` bash script by running `cd ../dell-csi-helm-installer && ./csi-install.sh --namespace vxflexos --values ../helm/myvalues.yaml`
+10. Install the driver using `csi-install.sh` bash script by running `cd ../dell-csi-helm-installer && ./csi-install.sh --namespace vxflexos --values ../helm/myvalues.yaml` or install helm charts without shell scripts by following README.md in helm folder.
```
[the remark this answers is not in the document]
Comment (nb950) [reply content/docs/csidriver/installation/helm/powerflex.md]: updated

**Review by jooseppi-luna — DISMISSED** [review]

**Review by yifeijin — APPROVED** [review]

**Comment by AronAtDell** [comment]
Do we need anything else for this to be merged?

**Review by jooseppi-luna — APPROVED** [review]
LGTM, thanks Jai

**Review by gallacher — APPROVED** [review]
LGTM

**Comment by nb950** [comment]
unverified commits

**Pull request closed without merging**